cowprotocol/accounting/slippage/slippage_breakdown_4059683.sql

```diff
@@ -0,0 +1,73 @@
+-- This query a breakdown of slippage on CoW Protocol
+--
+-- Parameters:
+--  {{start_time}} - the timestamp for which the analysis should start (inclusively)
+--  {{end_time}} - the timestamp for which the analysis should end (exclusively)
+--  {{blockchain}} - network to run the analysis on
+--
+-- The columns of the result are
+-- - block_time: time of settlement transaction
+-- - tx_hash: settlement transaction hash
+-- - token_address: address of token with slippage. contract address for erc20 tokens,
+--   0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee for native token
+-- - amount: signed value of slippage in atoms of the token; fees are represented as negative
+--   integers since they will be removed from imbalances
+-- - transfer_type: 'raw_imbalance' for imbalance observable on chain, 'protocol_fee' for the total
+--   protocol fee (including partner fee), 'network_fee' for network fees
+-- - price: USD price of one unit (i.e. pow(10, decimals) atoms) of a token
+-- - price_atom: USD price of one atom (i.e. 1. / pow(10, decimals) units) of a token
+-- - slippage_usd: USD value of slippage
+-- - slippage_native_atom: value of slippage in atoms of native token
+
+with raw_token_imbalances as (
+    select
+        block_time,
+        tx_hash,
+        token_address,
+        amount,
+        'raw_imbalance' as transfer_type,
+        date_trunc('hour', block_time) as hour --noqa: RF04
+    from "query_4021644(blockchain='{{blockchain}}',start_time='{{start_time}}',end_time='{{end_time}}')"
+),
+
+fee_balance_changes as (
+    select
+        block_time,
+        tx_hash,
+        token_address,
+        transfer_type,
```

Comment:
Same as comment above that i would expect the order of columns to be the same as the one used in `raw_token_imbalances`